lib/api/core/plugins/pluginsManager.js

```diff
@@ -222,8 +223,7 @@ class PluginsManager {
 
     const preparedPipes = [cb => cb(null, data, ...info)];
 
-    let i; // NOSONAR
-    for (i = 0; i < events.length; i++) {
+    for (let i = 0; i < events.length; i++) {
```

Comment:
Those changes doesn't have to exist only in 2-dev ? (with node10 support) ?

Reply:
Ooops

Reply:
test/bin/commands/start.test.js